spring-cloud-stream-binder-dapr-samples/azure-container-app-publisher-dapr/src/main/java/com/azure/spring/cloud/stream/binder/dapr/sample/DaprPublisherConfiguration.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.stream.binder.dapr.sample;
+
+import java.util.function.Supplier;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.messaging.Message;
+
+/**
+ * Dapr Publisher configuration.
+ */
+@Configuration
+@Profile("manual")
```

Comment:
Do we still need this`Configuration`? Besides, this way to publish messages should not be called as `manual`